psutil/_psutil_windows_uss.c

```diff
@@ -0,0 +1,73 @@
+/* -*- Mode: C++; tab-width: 8; indent-tabs-mode: nil; c-basic-offset: 2 -*- */
+/* vim: set ts=8 sts=2 et sw=2 tw=80: */
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "_psutil_windows_uss.h"
+#include <psapi.h>
+
+BOOL
+calc_uss(DWORD target, unsigned long long* aN)
+{
+  BOOL result = FALSE;
+  HANDLE proc;
+  PSAPI_WORKING_SET_INFORMATION tmp;
+  DWORD tmpSize = sizeof(tmp);
+  size_t entries, privatePages, i;
+  DWORD infoArraySize;
+  PSAPI_WORKING_SET_INFORMATION* infoArray;
+  SYSTEM_INFO si;
+
+  proc = OpenProcess(PROCESS_QUERY_INFORMATION | PROCESS_VM_READ,
+                     FALSE, target);
```

Comment:
You should use `psutil_handle_from_pid` defined in `psutil/arch/windows/process_info.c`; that will take care of raising `NoSuchProcess` and `AccessDenied` exception if needed.